pkgs/development/libraries/gstreamer/rtsp-server/default.nix

```diff
@@ -20,6 +20,12 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ];
 
+  # To use split outputs, we need this so double prefix won't be used in the
+  # pkg-config files
```

Comment:
Could you move this just before the patch. And also add a link to the merge request so we can track when this will become unnecessary.

Reply:
I didn't submit _my_ patch upstream, if that's what you meant. But I did move the comment right above the patch and I mentioned that 1 year old merge request that upstream haven't merged.

I got the impression that we don't suggest upstream such fixes for gst's plugins, as nearly every gst derivation has a `./fix-pkgconfig-includedir.patch` patch and non mention an upstream url for this patch.